projects/canopy/src/lib/modal/modal.stories.ts

```diff
@@ -28,7 +28,7 @@ export const standard = () => ({
       <lg-modal-header [headingLevel]="headingLevel">Lorem ipsum</lg-modal-header>
       <lg-modal-body>
         Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-        <lg-modal-body-timer timer="0:25"></lg-modal-body-timer>
+        <lg-modal-body-timer timer="90"></lg-modal-body-timer>
```

Comment:
Could you maybe update the notes to explain the type of input required please?

Reply:
updated